packages/core/src/utils/toCSS/toCSS.spec.ts

```diff
@@ -1,10 +1,6 @@
-import { describe, expect, it, jest } from '@jest/globals';
+import { describe, expect, it } from '@jest/globals';
 import { toCSS } from './toCSS';
 
-jest.mock('../../utils', () => ({
-  kebabCase: (s: string) => s.replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase(),
-}));
-
```

Comment:
By removing the mock we're testing `kebabCase` twice, once here and another in its own unit tests.

Which is fine, but this file is no longer strictly "unit" testing.

I'm personally ok with it in this instance but might be trickier in other cases.

Reply:
I agree that mocking is great in cases when you mock the output directly.

However, here it's mocking the actual utility.

Which IMO does not make sense - so same utility is written twice...

Reply:
We could rewrite the mock per test, so it doesn't mock the implementation, but the return.

But again, should be fine as is in this one case.